test/docker-compose.yml

```diff
@@ -73,3 +74,29 @@ services:
             retries: 2
             interval: 3s
             timeout: 2s
+    postgres-pixl-db:
+        build:
+            context: ../
+            dockerfile: ./docker/postgres/Dockerfile
+        environment:
+            POSTGRES_USER: ${PIXL_DB_USER}
+            POSTGRES_PASSWORD: ${PIXL_DB_PASSWORD}
+            POSTGRES_DB: ${PIXL_DB_NAME}
+            PGTZ: ${TZ:-Europe/London}
+        env_file:
+            - ../docker/common.env
```

Comment:
Imagine you've seen but from CI: `/home/runner/work/PIXL/docker/common.env: no such file or directory`